Core/src/org/sleuthkit/autopsy/casemodule/Case.java

```diff
@@ -1352,7 +1328,15 @@ public String getOutputDirectory() {
      * @return The temp subdirectory path.
      */
     public String getTempDirectory() {
-        return getOrCreateSubdirectory(TEMP_FOLDER);
+        // get temp folder scoped to the uuid of the case
```

Comment:
Is this comment accurate?

Reply:
I think `getName()` is a unique identifier.  I believe it is the case name and a time stamp.

Reply:
Yes it's the case name + timestamp, but I don't exactly consider that a UUID? Especially since you use an actual UUID in one of the other temp paths.

Reply:
Sure.  Change it from 'uuid' to 'unique identifier'?

Reply:
I think just "case name and timestamp" so there's no confusion

Reply:
Ok.  Should be pushed up now.